StyleCop.Analyzers/StyleCop.Analyzers/Helpers/NamedTypeHelpers.cs

```diff
@@ -91,5 +92,25 @@ internal static Location GetNameOrIdentifierLocation(SyntaxNode member)
             location = location ?? member.GetLocation();
             return location;
         }
+
+        /// <summary>
+        /// Returns whether or not a member is implementing an interface member.
+        /// </summary>
+        /// <remarks>
+        /// This method does only check the interfaces the containing type is implementing directly.
+        /// If a derived class is implementing an interface and this member is required for it
+        /// this method will still return false.
+        /// </remarks>
+        /// <param name="memberSymbol">The member symbol that should be analyzed.</param>
+        /// <returns>true if the member is implementing an interface member, otherwise false.</returns>
+        internal static bool IsImplementingAnInterfaceMember(ISymbol memberSymbol)
+        {
+            var typeSymbol = memberSymbol.ContainingType;
+
+            return typeSymbol != null && typeSymbol.AllInterfaces
```

Comment:
This is only called if the diagnostic would have been reported in the old version. So basicly in a project that follows this rule never.